### PR TITLE
Gradyear filter

### PR DESCRIPTION
**Pull request by tejalreddy**
Added a graduation year filter and made other misc changes to industry and tags dropdowns

### CODE REVIEW

File: client/components/Industry/IndustryButton.vue [hunk 1]
```diff
@@ -2,7 +2,9 @@
     <multiselect
         v-model="value"
         :options="options"
-        :show-labels="false"
+        :show-labels="true"
+        select-label="Click to select"
+        deselectLabel="Click to unselect"
```
Comment (hxliu33) [inline client/components/Industry/IndustryButton.vue:7]: - [ ] nit: standardize naming pascal case vs camel case

File: server/user/middleware.ts [hunk 2]
```diff
@@ -87,6 +87,28 @@ const isValidGraduationYear = (req: Request, res: Response, next: NextFunction)
...
+/**
+ * Checks if a email in req.params is valid
+ */
+ const isValidGraduationYearInParams = (req: Request, res: Response, next: NextFunction) => {
+  if (Number.isInteger(req.params.graduationYear)) {
+    res.status(400).json({
+      error: 'Graduation Year must be number.'
+    });
+    return;
+  }
```
Comment (hxliu33) [inline server/user/middleware.ts:99]: have we tested using grad year filtering yet to ensure this middleware works? i know we had some issues with this structure when we used it for creating a user

**Review by hxliu33 — CHANGES_REQUESTED** [review]
overall good, j some minor things

File: server/user/middleware.ts (hunk 2 above)
Comment (tejalreddy) [reply server/user/middleware.ts]: Fixed

**Review by jasmine2000 — APPROVED** [review]

**Pull request merged**